cmd/bootstrap/cmd/google.go

```diff
@@ -0,0 +1,89 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"cloud.google.com/go/storage"
+	"google.golang.org/api/iterator"
+	"google.golang.org/api/option"
+)
+
+// GoogleBucket ...
+type GoogleBucket struct {
```

Comment:
we could keep all these functions and structs private to this package since they are only being used in this package `cmd`.

Reply:
☝️ consider making these functions package private by renaming them to start with a lower case letter. e.g. `newGoogleucket`, `newClient`, `getFiles` etc.